app/src/main/res/values/strings.xml

```diff
@@ -927,4 +927,12 @@ Do you want to switch to this account?</string>
   <string name="self_hosted_server_url">Self-hosted server URL</string>
   <string name="passkey_operation_failed_because_user_could_not_be_verified">Passkey operation failed because user could not be verified.</string>
   <string name="user_verification_direction">User verification</string>
+  <string name="welcome_title_1">Privacy, prioritized</string>
+  <string name="welcome_message_1">Save logins, cards, and identities to your secure vault. Bitwarden uses zero-knowledge, end-to-end encryption to protect what’s important to you.</string>
+  <string name="welcome_title_2">Never guess again</string>
+  <string name="welcome_message_2">Set up biometric unlock and autofill to log into your accounts without typing a single letter.</string>
+  <string name="welcome_title_3">Level up your logins</string>
+  <string name="welcome_message_3">Use the generator to create and save strong, unique passwords for all your accounts.</string>
+  <string name="welcome_title_4">Your data, when and where you need it</string>
+  <string name="welcome_message_4">Save unlimited passwords across unlimited devices with Bitwarden mobile, browser, and desktop apps.</string>
```

Comment:
This deviates from the established pattern of resource keys and their value aligning. @vvolkgang @fedemkr do you have any concerns with us starting to use descriptive, context based resource keys like this instead? This approach has implications on Crowdin sync, since we would no longer be changing keys when their value is changed.

Aligning keys and values like we've been doing means we lose translations when the key+value changes.

Decoupling keys and values means we retain translations when values change, but also means we have the potential to continue showing old values in other languages until corresponding translations are updated.

Reply:
@shannon-livefront After some internal discussion we decided it's best to continue the same naming pattern for titles. Since the messages are so long we can keep them named as `welcome_message_n`.

Reply:
Thank you for the guidance!